### DISCUSSION TITLE
Disable Ukraine Flag in Display Colors

### DISCUSSION THREAD

**Discussion by DiablosGit** [tbnobody/OpenDTU#1680] [Q&A]
Hi,

I would like to have my Display only one color, at the moment it represents the Ukraine flag. I don't have a problem with the colors but its hard to read :)
I am using a ESP32 WROOM from ideals park with integrated display.

Please help me :)

Thx
![Uploading IMG_0273.JPG…]()

**Comment by stefan123t** [comment]
Hi @DiablosGit 

I could only find this image of an OLED with yellow/blue dots. This coloring is due to the hardware of the OLED display.

![image](https://github.com/tbnobody/OpenDTU/assets/11159753/6a97bfd2-adb8-4071-9fdb-210969833852)

There are multiple hardware versions of these displays in plain-white, plain-yellow or plain-blue and this "Ukraine flag" :upside_down_face: / yellow-and-blue-colored display variants.

![image](https://github.com/tbnobody/OpenDTU/assets/11159753/bd6ccb08-2f70-4054-b0ab-3ca7d2818092)

This is probably nothing we can change from the software side, you would rather need to get an other display module.

Could you maybe try to re-upload your image ?

You probably posted your comment, before the `Uploading status` became replaced with the right image tag after upload completed: e.g.
`![image](https://github.com/tbnobody/OpenDTU/assets/11159753/6a97bfd2-adb8-4071-9fdb-210969833852)`
